turbo/reference/streams.md

```diff
@@ -2,6 +2,7 @@
 order: 3
 title: Turbo ストリーム
 description: Turbo ストリームのリファレンス
+commit: "e9e0628"
```

Comment:
The translated PR was created on Jan 23, 2024.
Judging from this data, the commit might be e9e0628.
ref: https://github.com/everyleaf/hotwire_ja/pull/30
ref: https://github.com/hotwired/turbo-site/commits/main/_source/reference/streams.md